test/e2e/flowaggregator_test.go

```diff
@@ -1058,12 +1130,12 @@ func deletePerftestServices(t *testing.T, data *TestData) {
 	}
 }
 
-// getBandwidthAndSourcePort parses iperf commands output and returns bandwidth
-// and source port. Bandwidth is returned as a slice containing two strings (bandwidth
-// value and bandwidth unit).
-func getBandwidthAndSourcePort(iperfStdout string) ([]string, string) {
+// getBandwidthAndPort parses iperf commands output and returns bandwidth,
+// source port and destination port. Bandwidth is returned as a slice containing
+// two strings (bandwidth value and bandwidth unit).
+func getBandwidthAndPort(iperfStdout string) ([]string, string, string) {
```

Comment:
should be `getBandwidthAndPorts` if you return both the source and destination ports

Reply:
done